pkg/endpointmanager/idallocator/allocator_test.go

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2018 Authors of Cilium
 
+// +build !privileged_tests
+
```

Comment:
To avoid these errors and the following fixup, couldn't we detect the end of the license comment instead of replacing the entire comment block?

Reply:
This issue only occurs when the build flag is not separated from the license comment block, its actually a unlikely case other then a handful of files Andre has pointed out.

Reply:
Will any tests fail if we missed one of these incorrectly removed tags? I.e., do we have a way to ensure we didn't miss any?

Reply:
There are CI jobs which check for tags like: https://github.com/cilium/cilium/blob/master/contrib/scripts/check-missing-tags-in-tests.sh

Whether they will catch call issues, not too sure, weve been scripting the check out of band via: https://cilium.slack.com/archives/C2B917YHE/p1626209673481600

Reply:
Looks covered then! Fine by me as is in that case :ship: